src/client-apps/nuxeo-drive/nuxeo-drive-faq/nuxeo-drive-architecture-recommendations-windows.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Nuxeo Drive Configuration Requirements on Windows
+review:
+  comment: ''
+  date: '2024-10-05'
+  status: ok
+labels:
+  - nuxeo-drive
+  - mlumeau
+  - multiexcerpt
+  - excerpt
+  - lts2023-ok
+toc: true
+tree_item_index: 100
+---
+
+## Before You Install
+
+### Supported Windows Version 
+
+- Windows >= 10, both 32 and 64 bits
+
+### Nuxeo Drive Version 
+
+- Use Latest Nuxeo Drive version 
+  
+### CPU and RAM Requirements 
+
+It depends on the third-party application used for file editing as well as the size of the file. Nuxeo Drive is tested against Microsoft Office 365 and we recommend using it.
+To use Direct Edit, you should allocate at least **500MB of free memory** for Nuxeo Drive on top of Microsoft Office consumption for direct edit usage or at least **twice the file size** for larger files.
+
+### Permissions 
+
+Administrator rights are required on the machine while installing or uninstalling the application. Upgrade is similar to installation, so similar level of access is required.  
+
+### Dependencies 
+
+- Nuxeo-drive version 5.5.0 => Python v3.9.5 is compatible and required to be installed separately in the local machine. 
+=> Make sure that no other software overrides the supported python version. 
+
+Nuxeo Drive embeds its own dependencies, you should ensure that you do not override these dependencies system wide to prevent issues. 
```

Comment:
```suggestion
Although Nuxeo Drive embeds its own dependencies, you should ensure that you do not override these dependencies system wide to prevent issues. 
```